### PR TITLE
Adopt Contributor Covenant

### PR DESCRIPTION
**Pull request by zpao**
In order to foster healthy open source communities, we're adopting the
[Contributor Covenant](https://www.contributor-covenant.org/). It has been
built by open source community members and represents a shared understanding of
what is expected from a healthy community.

### CODE REVIEW

**Comment by MylesBorins** [comment]
Hey all. Glad to see that this landed. Over in Node.js we found that without explicit moderation guidelines a Code of Conduct can actually be harmful 😅... especially if folks are spending time debating out how to enforce the code of conduct in the midst of a time sensitive matter.

We implemented [moderation guidelines](https://github.com/nodejs/admin/blob/master/Moderation-Policy.md) which have been extremely helpful, especially in empowering collaborators to know what actions they can take. Things being ambiguous generally leads to inaction

We also implemented [membership expectations](https://github.com/nodejs/admin/blob/master/MemberExpectations.md) that go above and beyond the code of conduct for collaborators and leadership. 

Happy to answer any questions or hop on a call if you want to discuss.

**Comment by diomawuntu** [comment]
Halo 
Im Request  hack number whatsapp

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -1,3 +1,76 @@
 # Code of Conduct
 
-Facebook has adopted a Code of Conduct that we expect project participants to adhere to. Please [read the full text](https://code.fb.com/codeofconduct/) so that you can understand what actions will and will not be tolerated.
```
Comment (ws1117f) [inline CODE_OF_CONDUCT.md]: hi

File: CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -1,3 +1,76 @@
 # Code of Conduct
```
Comment (ws1117f) [inline CODE_OF_CONDUCT.md]: hy

**Pull request merged**